PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -203,94 +203,113 @@ fi
+           or={i,j,k,i-,j-,k-}        (Used with either a gradient echo field map 
+                                      or a spin echo field map)
+  --gdcoeffs=<file path>              File containing gradient distortion
+                                      coefficients, Set to "NONE" to turn off
+  --avgrdcmethod=<avgrdcmethod>       Averaging and readout distortion correction
+                                      method. See below for supported values.
+
+    "${NONE_METHOD_OPT}"
+     average any repeats with no readout distortion correction
+
+    "${FIELDMAP_METHOD_OPT}"
+       equivalent to "${SIEMENS_METHOD_OPT}" (see below)
+       SiemensFieldMap is preferred. This option value is maintained for
+       backward compatibility.
+
+    "${SPIN_ECHO_METHOD_OPT}"
+       average any repeats and use Spin Echo Field Maps for readout
+       distortion correction
+
+    "${GENERAL_ELECTRIC_METHOD_OPT}"
+       average any repeats and use General Electric specific Gradient
+       Echo Field Maps for readout distortion correction
+
+    "${SIEMENS_METHOD_OPT}"
+       average any repeats and use Siemens specific Gradient Echo
+       Field Maps for readout distortion correction
+
+  --topupconfig=<file path>           Configuration file for topup or "NONE" if not used
+  [--bfsigma=<value>]                 Bias Field Smoothing Sigma (optional)
+  [--custombrain=(NONE|MASK|CUSTOM)]  If PreFreeSurfer has been run before and you have created a custom
+                                      brain mask saved as "<path>/T1w/custom_acpc_dc_restore_mask.nii.gz", specify "MASK". 
+                                      If PreFreeSurfer has been run before and you have created custom structural images, e.g.: 
+                                      - "<path>/T1w/T1w_acpc_dc_restore_brain.nii.gz"
+                                      - "<path>/T1w/T1w_acpc_dc_restore.nii.gz"
+                                      - "<path>/T1w/T2w_acpc_dc_restore_brain.nii.gz"
+                                      - "<path>/T1w/T2w_acpc_dc_restore.nii.gz"
+                                      to be used when peforming MNI152 Atlas registration, specify "CUSTOM".
+                                      When "MASK" or "CUSTOM" is specified, all the steps until Atlas registration 
+                                      are skipped.
+                                      If the parameter is omitted or set to NONE (the default), 
+                                      standard image processing will take place.
+  [--mpp-mode=(HCPStyleData|          Which variant of MPP to use. "HCPStyleData" (the default) follows the processing steps
+               LegacyStyleData)]      described in Glasser et al. (2013) and requires 'HCP-Style' data acquistion. 
+                                      "LegacyStyleData" allows additional processing functionality and use of some acquisitions 
+                                      that do not conform to 'HCP-Style' expectations:
```

Comment:
"i.e., in this case:"

Reply:
Changed.

Reply:
"i.e., in this case" is basically redundant with itself.  I would prefer phrasing of the form "...allows the use of these non-HCP-conformant settings:"